precompiles/proxy/Proxy.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.0;
+
+/// @author The Moonbeam Team
+/// @title The interface through which solidity contracts will interact with the Proxy pallet
+/// @custom:address 0x000000000000000000000000000000000000080b
+interface Proxy {
+    /// @dev Defines the proxy permission types that may be combined via `|` operator
```

Comment:
I don't think types are supposed to be combined with `|`, as in our runtimes they follow indicies 0, 1, 2, ... instead of 1, 2, 4, ... for bitflags.
https://github.com/PureStake/moonbeam/blob/master/runtime/moonriver/src/lib.rs#L794